cgeglio/rancid-tomatillos
# Login functionality

### What does this PR do?
- Adds functionality to the login page: if a user logs in, they will be redirected to the homepage
- Adds error messaging in case the user does not fill out an email or password
- Adds the user (with id from post request) to state once they are logged in

### Where should the reviewer start?
- LoginForm.js

### How should this be manually tested?
- Login using our default email/password, you should be redirected to the homepage.
- Try logging in without an email/password and you should see an error message

### Potential changes?
- Need to update the error messaging to not log the user in if the post request is not completed, rn it just checks that the fields are filled out

### Relevant issues?
#3
